docs/guidance/ingress.md

```diff
@@ -1,10 +1,16 @@
 ## Ingress Usage
 
+* KubeRay built-in ingress support: KubeRay will help users create a Kubernetes ingress when `spec.headGroupSpec.enableIngress: true`. Currently, the built-in support only supports simple Nginx setups. Note that users still need to install ingress controller by themselves. **We do not recommend users use built-in ingress support in a production environment with complex routing requirements.**
```

Comment:
nit: "users use" is a little redundant
Also, I generally prefer not to use the word "user" in documentation when possible. I think it's usually to either use "you" or to not directly refer to the reader/user at all. 

```suggestion
* KubeRay built-in ingress support: KubeRay will help users create a Kubernetes ingress when `spec.headGroupSpec.enableIngress: true`. Currently, the built-in support only supports simple Nginx setups. Note that users still need to install ingress controller by themselves. **We do not recommend using built-in ingress support in a production environment with complex routing requirements.**
```